### PR TITLE
[WIP] Fix junos integration zuul CI failures (#57309)

### PR DESCRIPTION
**Pull request by danielmellado**
##### SUMMARY
Fixes junos zuul issues, backported from Devel

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
junos_facts

### CODE REVIEW

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now a6d1983415fa8c2bb632a7f3f75f0858a6e5a69c

File: changelogs/fragments/junos-ci-fix-d533a54d99180980.yaml
Comment (abadger) [inline changelogs/fragments/junos-ci-fix-d533a54d99180980.yaml]: changelogs are for the end users.  So they should reflect what was fixed for them.  Tracebacks?  Idempotency?  Behaviour that didn't match the documentation?

Changes to tests don't need to be changeloged but the change to lib/ansible/plugins/netconf/__init__.py does.

File: lib/ansible/plugins/netconf/__init__.py
Comment (pabelanger) [inline lib/ansible/plugins/netconf/__init__.py:226]: We recently found this breaks on iosxr, so we need to hold off on this backport until we fix this on devel

**Review by pabelanger — CHANGES_REQUESTED** [review]
-1

**Comment by abadger** [comment]
I've marked this work in progress for now.  @danielmellado feel free to change the title and remove WIP once this is ready for me to look at again.

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now 5f79387df457aed0c4cb4a1e832efa60c89c6860

**Comment by danielmellado** [comment]
I've updated the fragment, however, let's hold on merging this until we check @pabelanger comments about this breaking iosxr.

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now 0f7d24c60fa393a35b504dede2ba0cf00260a7a6

**Review by pabelanger — CHANGES_REQUESTED** [review]
This is not correct, we are removing the logic that will make both junos / iosxr work.

**Comment by abadger** [comment]
Tagged as WIP again.  @danielmellado it would help me if you wait on removing WIP until after you're ready for me to merge it (ie: if you need someone else to review it first, let them review it before reviewing WIP).  If that interferes with your workflow with the potential reviewers, let me know and we'll figure out who should gate on WIP and who shouldn't.

**Comment by danielmellado** [comment]
no longer needed

**Pull request closed without merging**
